Project.toml

```diff
@@ -40,32 +40,32 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
-Calculus = "0.4,0.5"
-Colors = "0.11,0.12"
-Conda = "1"
+Calculus = "^0.5"
+Colors = "^0.12"
+Conda = "^1"
```

Comment:
The "^" is default behavior, so adding the caret when there isn't another operator there doesn't change behavior.

Reply:
I dropped this commit!